specs/trust-store-trust-policy.md

```diff
@@ -53,31 +53,67 @@ $XDG_CONFIG_HOME/notation/trust-store
 The Trust store currently supports three kinds of identities, additional identities may be supported in future :
 
 - **Certificates**: The `x509/ca` trust store contains named stores that contain Certificate Authority (CA) root certificates.
-- **SigningAuthority Certificate**: The `x509/signingAuthority` trust store contains named stores that contain Siging Authority's root certificates.
+- **SigningAuthority Certificate**: The `x509/signingAuthority` trust store contains named stores that contain Signing Authority's root certificates.
 - **Timestamping Certificates**: The `x509/tsa` trust store contains named stores with Time Stamping Authority (TSA) root certificates.
 
-Any additional sub directories under names stores and certificates in it are ignored. **NOTE**: Implementation SHOULD warn if it finds sub directories with certificates under a named store, to help diagnose misconfigured store.
+Any additional sub directories under a named store and certificates in it are ignored. **NOTE**: Implementation SHOULD warn if it finds sub directories with certificates under a named store, to help diagnose misconfigured store.
 
 ## Trust Policy
 
-Users who consume signed artifact from a registry use the trust policy to specify trusted identities which will sign the artifacts, and level of signature verification to use.
+Users who consume signed artifacts from OCI registries, or signed arbitrary blobs stored on file system, use trust policies to specify trusted identities that signed the artifacts, and level of signature verification to enforce. Trust policy files are JSON documents.
 
-### Trust Policy Schema
+**NOTE**: Users are required to setup separate trust policy files for verifying different types of signed artifacts. Notation expects OCI trust policy file to be placed at `{NOTATION_CONFIG}/trustpolicy.oci.json`. For backward compatibility reasons, Notation accepts `{NOTATION_CONFIG}/trustpolicy.json` as OCI trust policy file as well. For arbitrary blobs, Notation expects trust policy file to be placed at `{NOTATION_CONFIG}/trustpolicy.blob.json`.
```

Comment:
Can we just keep the original `{NOTATION_CONFIG}/trustpolicy.json` for OCI related artifacts? So the two policy files are:
`{NOTATION_CONFIG}/trustpolicy.json` and `{NOTATION_CONFIG}/trustpolicy.blob.json`.

Reply:
> Original `{NOTATION_CONFIG}/trustpolicy.json` is still allowed. Are you suggesting to remove `{NOTATION_CONFIG}/trustpolicy.oci.json`?

@rgnote Yes, because we are not making any change to the original oci trust policy. Implementations then don't need to make any update regarding the oci trust policy, it's the old content with the old name.

Reply:
It can be confusing to some users who do both OCI and blob verification as to why trustpolicy.json is for OCI. They can use `{NOTATION_CONFIG}/trustpolicy.oci.json` and `{NOTATION_CONFIG}/trustpolicy.blob.json` for disambiguation.

Reply:
I see. SGTM, please feel free to resolve the comment.